packages/components/src/components/link/_link.scss

```diff
@@ -60,7 +60,10 @@
     color: $visited-link;
   }
 
-  .#{$prefix}--link--disabled {
```

Comment:
while I disagree with the usage of using a `<p>` for a disabled link (it's not correct semantically to screen reader users). I do think that this selector is fine to keep because I have seen demos that did not include the `bx--link` class with the disabled class and this would help cover any of those stragglers out there.